wagtail/images/templates/wagtailimages/images/url_generator.html

```diff
@@ -35,8 +35,25 @@
         {% endpanel %}
 
         {% trans "URL" as heading %}
-        {% panel id="url" icon="link" heading=heading %}
-            <textarea class="w-image-url-generator__url" id="result-url" aria-labelledby="url-heading" rows="1" data-controller="w-action" data-action="focus->w-action#select"></textarea>
+        {% panel id="url" heading=heading %}
+            <div data-controller="w-clipboard w-clone" role="group" aria-labelledby="clipboardGroupLabel" data-w-clone-clear-delay-value="3500" data-action="w-clipboard:start->w-clone#clear w-clipboard:copied->w-clone#add w-clipboard:error->w-clone#add">
+                <label id="clipboardGroupLabel" hidden>Clipboard Actions</label>
+
+                <textarea class="w-image-url-generator__url" id="result-url" rows="1" data-controller="w-action" data-action="focus->w-action#select" data-w-clipboard-target="value" aria-label="Image URL">
+                </textarea>
+
+                <button type="button" class="button button-secondary" data-action="w-clipboard#copy" aria-label="Copy Image URL">
+                    Copy URL
```

Comment:
```suggestion
                    {% trans 'Copy URL' %}
```
A reminder to translate this  one too please.